samples/dotnet/kernel-syntax-examples/Skills/JiraSkill/README.md

```diff
@@ -0,0 +1,5 @@
+# Jira Open API Schema
+We have our own curated version of the jira open api schema because the one available online 
+at https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/certified-connectors/JIRA/apiDefinition.swagger.json, 
+doesn't follow OpenAPI specification for all of its operations. For example CreateIssueV2, its body param does not describe properties 
```

Comment:
did you report it to their repo?

Reply:
No I have not. The PowerPlatform Connector repo uses an older version of the swagger.json, version 2. Atlassian has a version 3 here: https://developer.atlassian.com/cloud/jira/platform/swagger-v3.v3.json. And version 3 has fixed this but its a more dynamic building of the schema for operations. I don't believe we support that in the kernel yet (I think Sergey sergey was talking about how to pass information like this more generically as jsons rather than as specific Context Variables which support limited data types).